.github/scripts/config.sh

```diff
@@ -35,6 +35,7 @@ TOOLCHAIN_PATH=${TOOLCHAIN_PATH:-~/cross-$TOOLCHAIN_NAME}
 TOOLCHAIN_FILE=${TOOLCHAIN_FILE:-$PWD/.github/cmake/$TARGET.cmake}
 TOOLCHAIN_PACKAGE_NAME=${TOOLCHAIN_PACKAGE_NAME:-$TOOLCHAIN_NAME-toolchain.tar.gz}
 DEJAGNU_FILE=${DEJAGNU_FILE:-$PWD/.github/scripts/toolchain/site.exp}
+ROOT_DIR=${ROOT_DIR:-`dirname ${BASH_SOURCE[0]}`}
```

Comment:
The default should be either `$PWD` or `` `dirname ${BASH_SOURCE[0]}`/../..``.

Reply:
I'm not sure what you mean here, can you elaborate please? Do you suggest to set `ROOT_DIR` to `$PWD` or  `dirname ${BASH_SOURCE[0]}/../..` like:
```
ROOT_DIR=${ROOT_DIR:-`dirname ${BASH_SOURCE[0]}/../..`}
```
or are you talking about the check "use ROOT_DIR if it was previously set"?